spynnaker/pyNN/models/neuron/master_pop_table.py

```diff
@@ -151,33 +191,31 @@ def get_exact_master_population_table_size(
 
         # Multiply by 2 to get an upper bound
         return (
-            (n_vertices * 2 * _MasterPopEntry.MASTER_POP_ENTRY_SIZE_BYTES) +
-            (n_entries * 2 * _MasterPopEntry.ADDRESS_LIST_ENTRY_SIZE_BYTES) +
+            (n_vertices * 2 * _MASTER_POP_ENTRY_SIZE_BYTES) +
+            (n_entries * 2 * _ADDRESS_LIST_ENTRY_SIZE_BYTES) +
             8)
 
     def get_allowed_row_length(self, row_length):
```

Comment:
this method would be better with row_length -1 as a seperate param. and it has a magic number of 256. reminder of the 256 constant tie in

Reply:
Fixed